konduit-serving-api/src/main/java/ai/konduit/serving/pipeline/PipelineStep.java

```diff
@@ -135,7 +135,7 @@ public PipelineStep input(String[] columnNames, SchemaType[] types) throws Excep
      * @return this pipeline step
      * @throws Exception key error
      */
-    public PipelineStep setInput(String inputName, String[] columnNames, SchemaType[] types)
+    public PipelineStep input(String inputName, String[] columnNames, SchemaType[] types)
```

Comment:
I changed this because `input` is a protected keyword in python and I wanted to keep things consistent with `setInput` and `set_input` on the Java and Python sides respectively.

Reply:
Sure that makes sense. I updated it to the set* signature https://github.com/KonduitAI/konduit-serving/pull/36/commits/c9232f657779764d89447dd8397da154f11991d1